include/fi.h

```diff
@@ -200,6 +198,12 @@ int fi_rma_target_allowed(uint64_t caps);
 
 #define DEFAULT_ABI "FABRIC_1.0"
 
+#ifdef  HAVE_ALIAS_ATTRIBUTE
+#define default_symver_pre(a) a##_
+#else
+#define default_symver_pre(a) a
```

Comment:
I wonder if `default_symver_pre` (and `default_symver`) should become all-caps (i.e., `DEFAULT_SYMVER_PRE` and `DEFAULT_SYMVER`).  I realize that this is a *little* out of scope of this PR, but `default_symver_pre` is new, so I guess it's fair game to bring it up.  :-)

I ask because the convention of all-caps preprocessor macros is a good one.  When I was reading other parts of this patch, I saw `default_symver_pre` and wondered "is that some kind of universal macro on Linux and OS X?" No, it's not -- it's defined here in libfabric.

Just my $0.02...